cli/config/configuration.toml.in

```diff
@@ -167,6 +167,13 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # all practical purposes.
 #entropy_source= "@DEFENTROPYSOURCE@"
 
+# If set to an absolute path within the guest rootfs, the agent will search
+# this directory for OCI hooks and add them to the guest container's lifecycle.
```

Comment:
Please emphasize the fact that the hooks are executed in the guest rather than in the host.

Reply:
You could also add a comment along the lines of:

> This is an advanced option. Since OCI hooks are always run on the host, most users can leave this variable unset.

Reply:
@jodh-intel

> Since OCI hooks are always run on the host

Did you mean "guest"? Or are you referring to OCI hooks from the original OCI runtime spec?